package.json

```diff
@@ -31,8 +31,8 @@
         "geolib": "^3.2.1",
         "localized-countries": "^2.0.0",
         "lucene-escape-query": "^1.0.1",
-        "mjolnir.js": "^2.4.0",
         "mapbox-gl": "^3.0.0",
+        "mjolnir.js": "^2.4.0",
```

Comment:
why this move ?

Reply:
npm automatically sort dependencies list